Assets/Scripts/Collectable.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Collectable : MonoBehaviour {
+
+    [SerializeField]
+    private float rotationSpeed;
+
+	private void FixedUpdate ()
+    {
+        transform.Rotate(Vector3.up * rotationSpeed * Time.deltaTime);
+	}
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        if (collision.CompareTag("Player"))
+        {
```

Comment:
Add sound effect to collectible, make sure you aren't destroying the collectible too early. Watch and follow my tutorial: https://youtu.be/GTsilz2v_PM